File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/goodsDescService.js
app.service('goodsDescService', function ($http) {
    //旧的查询方法(弃置)								   
    this.findPage = function (pageNum, pageSize) {
        return $http.get('../goodsDesc/findPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize)
    };

    //查询方法,读取列表数据绑定到表单中
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post('../goodsDesc/findPage.do?pageNum=' + pageNum + "&pageSize=" + pageSize, searchEntity)
    };

    //删除方法
    this.removeGoodsDescList = function (idList) {
        return $http.get('../goodsDesc/removeGoodsDesc.do?idList=' + idList)
    };

    //根据id查询
    this.findOne = function (id) {
        return $http.get('../goodsDesc/findOne.do?id=' + id)
    };

    //修改品牌
    this.updateGoodsDesc = function (goodsDesc) {
        return $http.post('../goodsDesc/updateGoodsDesc.do', goodsDesc)
    };

    //新增品牌
    this.saveGoodsDesc = function (goodsDesc) {
        return $http.post('../goodsDesc/saveGoodsDesc.do', goodsDesc)
    };   
});

File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/GoodsDescService.java
package com.pinyougou.sellergoods.service;
import java.util.List;
import com.pinyougou.pojo.TbGoodsDesc;							  
import entity.PageResult;
import entity.Result;
/**
 * goodsDesc服务层接口
 * @author jieweili
 *
 */
public interface GoodsDescService {

    /**
     * 返回全部列表(已弃置)
     * @return
     */
    List<TbGoodsDesc> findAll();
 
    /**
     * 返回分页对象(过时)
     * @param pageNum
     * @param pageSize
     * @return
     */
	 
    PageResult findPage(Integer pageNum, Integer pageSize);

    /**
     * 添加数据
     * @param goodsDesc
     * @return
     */
    public Result saveGoodsDesc(TbGoodsDesc goodsDesc);

    /**
     * 根据id查询一个对象
     * @param id
     * @return
     */
    TbGoodsDesc getGoodsDescById(Long id);

    /**
     * 修改对象
     * @param goodsDesc
     * @return
     */
    Result updateGoodsDesc(TbGoodsDesc goodsDesc);

    /**
     * 批量删除
     * @param idList
     * @return
     */
    Result removeGoodsDesc(Long[] idList);

    /**
     * 根据条件分页查询
     * @param goodsDesc
     * @param pageNum
     * @param pageSize
     * @return
     */
    PageResult listGoodsDescByExample(TbGoodsDesc goodsDesc, Integer pageNum, Integer pageSize);
 
}
